subprojects/android/android.gradle

```diff
@@ -0,0 +1,8 @@
+ext {
+    artifactName = 'mockito-android'
+    bintrayAutoPublish = false
+    mavenCentralSync = false
+}
+
+apply from: "$rootDir/gradle/java-library.gradle"
+apply from: "$rootDir/gradle/publishable-java-library.gradle"
```

Comment:
Android subproject is not there yet, but if we want correct pom we'd need to add that, right ?

```
dependency {
    runtime "org.mockito:mockito-core:$version"
    runtime "net.bytebuddy:byte-buddy-android:1.5.5"
}
```